tools/functional-tester/etcd-tester/tester.go

```diff
@@ -90,18 +90,19 @@ func (tt *tester) runLoop() {
 			log.Printf("etcd-tester: [round#%d case#%d] canceled stressers", i, j)
 
 			log.Printf("etcd-tester: [round#%d case#%d] checking current revisions...", i, j)
-			ok := false
+			connOk, ok := true, false
```

Comment:
i think we should move connOK into the for loop at line 95. Also we chould probably just use err at line 97

```
revs, rerr := tt.cluster.getRevision()
...
if rerr != nil {
    continue
}
```